content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Mengatur Batas Minimum dan Maksimum Memori pada sebuah Namespace
+content_type: task
+weight: 30
+---
+
+
+<!-- overview -->
+
+Laman ini menunjukkan cara untuk mengatur nilai minimum dan maksimum memori yang digunakan oleh Kontainer
+yang berjalan pada sebuah Namespace. Kamu dapan menentukan nilai minimum dan maksimum memori pada objek
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core). Jika sebuah Pod tidak memenuhi batasan yang ditentukan oleh LimitRange,
+Pod tersebut tidak dapat dibuat pada Namespace tersebut.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Tiap Node dalam klastermu harus memiliki setidaknya 1 GiB memori.
+
+
+
+
+<!-- steps -->
+
+## Membuat sebuah Namespace
+
+Buat sebuah Namespace sehingga sumber daya yang kamu buat pada latihan ini
+terisolasi dari komponen lain pada klatermu.
```

Comment:
typo?
```suggestion
terisolasi dari komponen lain pada klastermu.
```

Reply:
Sorry, I made awfully lot of typo for that word. Fixed in e323b8c5920d6ae02c050306495e784b8e19ad9f